compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/MetalRedrawer.kt

```diff
@@ -302,13 +307,27 @@ internal class MetalRedrawer(
                 return@autoreleasepool
             }
 
+            // Perform timestep and record all draw commands into [Picture]
+            pictureRecorder.beginRecording(Rect(
+                left = 0f,
+                top = 0f,
+                width.toFloat(),
+                height.toFloat()
+            )).also { canvas ->
+                canvas.clear(Color.WHITE)
```

Comment:
BTW Why do we keep `WHITE` here? Can it be transparent like in other platforms?

Reply:
We can try, but we'd better not hint toward transparent Compose views, because of alpha-blending problems I've mentioned earlier.